website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_position: 1
+description: Create one or more indexes on a collection to improve query performance and enable certain query features.
+---
+
+# `createIndexes`
+
+import CodeBlock from '@theme/CodeBlock';
+import CreateIndex from '!!raw-loader!./create-indexes/1-create-simple-index.request.js';
+import CreateUniqueIndex from '!!raw-loader!./create-indexes/2-create-unique-index.request.js';
+import CreateCompoundIndex from '!!raw-loader!./create-indexes/3-create-compound-index.request.js';
+import CreatePartialIndex from '!!raw-loader!./create-indexes/4-create-partial-index.request.js';
+import CreateSparseIndex from '!!raw-loader!./create-indexes/5-create-sparse-index.request.js';
+
+<!-- Category for Command "Administrative, Diagnostic, and Utility Commands" should be organized per folder as present in the compatibility document -->
+
+<!-- Short description of the command and its usage. -->
+
+Create one or more indexes on a given collection to improve query performance or enable certain query features (e.g., full-text, vector, and TTL indexes).
+
+<!-- Syntax structure of the command here if there are no unique variations, types, or usage examples to command. Otherwise, use syntax inside subcategories-->
+
+<!-- markdownlint-disable MD033 -->
+
+This command accepts a collection name and an array of index specification documents.
+The index specification document specifies the fields to be indexed, the index name, and other options such as uniqueness, partial filter expressions, and expiration times for TTL indexes.
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>',
+  indexes: [
+    {
+      key: {
+        '<field1>': <index_type>, // 1 for ascending, -1 for descending
+        '<field2>': <index_type> // optional
+      },
+      name: '<index_name>',
+      unique: <boolean>, // optional
+      partialFilterExpression: {
+        '<field>': { '<operator>': '<value>' }
+      }, // optional
+      sparse: <boolean> // optional
+    }
+  ]
+})
```

Comment:
![CleanShot 2025-05-26 at 23 32 43@2x](https://github.com/user-attachments/assets/c99d6ed7-fe01-461d-9c1f-72149b802d93)

Those unaligned comments don't look good. Let's drop them; the information is in the table anyway.